CRM/Core/Permission/WordPress.php

```diff
@@ -58,7 +58,20 @@ public function check($str) {
       return TRUE;
     }
 
-    require_once ABSPATH . WPINC . '/pluggable.php';
+	//CRM-15629
+	// During some extern/* calls we don't bootstrap CMS hence
+	// below constants are not set. In such cases, we don't need to
+	// check permission, hence directly return TRUE
+	if ( ! defined( 'ABSPATH' ) || ! defined( 'WPINC' ) ) {
+		  return TRUE;
+	}
+	if ( ! defined( ABSPATH ) || ! defined( WPINC ) ) {
```

Comment:
Will this line ever be reached, given the logic of line 65?

Also, shouldn't the code read `if ( ! defined( 'ABSPATH' ) || ! defined( 'WPINC' ) ) {`? Otherwise it'll throw PHP notices